spec/features/user/email_activation_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe "As a visitor", :vcr do
+  before :each do
```

Comment:
Add spec to show that different views are seen on the dashboard depending on if a user has confirmed with their email